Makefile

```diff
@@ -25,3 +25,21 @@ coverage:
 coveralls: coverage
 	go get github.com/mattn/goveralls
 	goveralls -coverprofile=$(COVERAGE_FILE) -service=github
+
+DURATION ?= 3s
+
+.PHONY: bench
+bench:
+	go clean -testcache
+	go test -bench=. -benchmem -benchtime=${DURATION}
+
+.PHONY: ab-perf
+ab-perf:
+	rm -rf bench
+	mkdir bench
+	go clean -testcache
+	go test -bench=. -benchmem -benchtime=${DURATION} > bench/b.txt
```

Comment:
Add `bench` directory to gitignore please. It is removed below, but for cases when something goes wrong it is better to hide a directory for Git.

Reply:
Already there.